app/models/concerns/roles.rb

```diff
@@ -8,7 +8,11 @@ module Roles
     klass.validate :roles_valid
 
     klass::ROLES.each do |role|
-      klass.scope role.pluralize.to_sym, -> { matching_role_query([role.to_s]) }
+      if role.split('_').last.numeric?
```

Comment:
can leave as is or could also do it by overriding/patching inflections, does become agfs --> agfss - i guess not

```
ActiveSupport::Inflector.inflections do |inflect|
  inflect.irregular 'agfs_scheme_9', 'agfs_scheme_9'
end
```

not sure if you can do this 'locally' or whether you have to use `config/initializers/inflections.rb`